src/shogun/evaluation/CrossValidation.cpp

```diff
@@ -172,8 +168,8 @@ float64_t CCrossValidation::evaluate_one_run(
 			/* evtl. update xvalidation output class */
 			CrossValidationFoldStorage* fold = new CrossValidationFoldStorage();
 			SG_REF(fold)
-			fold->set_run_index(index);
-			fold->set_fold_index(i);
+			fold->put("current_run_index", (index_t)index);
```

Comment:
maybe remove "current_" prefix?
Sorry, I am all in favour of removing redundancy from names these days :)